new_client/src/components/resources/ResourcePage.jsx

```diff
@@ -0,0 +1,197 @@
+import React from "react";
+import {
+  Box,
+  Container,
+  Grid,
+  ThemeProvider,
+  Typography,
+  createMuiTheme,
+  responsiveFontSizes,
+} from "@material-ui/core";
+
+import Topic from "./Topic";
+
+const topics = [
+  {
+    name: "Data Structures and Algorithms",
+    resources: [
+      {
+        name: "ABC",
+        link: "https://www.geeksforgeeks.org/",
+      },
+      {
+        name: "DEF",
+        link: "https://www.geeksforgeeks.org/",
+      },
+      {
+        name: "GHI",
+        link: "https://www.geeksforgeeks.org/",
+      },
+      {
+        name: "ABC",
+        link: "https://www.geeksforgeeks.org/",
+      },
+      {
+        name: "ABC",
+        link: "https://www.geeksforgeeks.org/",
+      },
+      {
+        name: "ABC",
+        link: "https://www.geeksforgeeks.org/",
+      },
+    ],
+  },
+  {
+    name: "Web Development",
+    resources: [
+      {
+        name: "ABC",
+        link: "https://www.geeksforgeeks.org/",
+      },
+      {
+        name: "DEF",
+        link: "https://www.geeksforgeeks.org/",
+      },
+      {
+        name: "GHI",
+        link: "https://www.geeksforgeeks.org/",
+      },
+      {
+        name: "ABC",
+        link: "https://www.geeksforgeeks.org/",
+      },
+      {
+        name: "ABC",
+        link: "https://www.geeksforgeeks.org/",
+      },
+      {
+        name: "ABC",
+        link: "https://www.geeksforgeeks.org/",
+      },
+    ],
+  },
+  {
+    name: "App Development",
+    resources: [
+      {
+        name: "ABC",
+        link: "https://www.geeksforgeeks.org/",
+      },
+      {
+        name: "DEF",
+        link: "https://www.geeksforgeeks.org/",
+      },
+      {
+        name: "GHI",
+        link: "https://www.geeksforgeeks.org/",
+      },
+      {
+        name: "ABC",
+        link: "https://www.geeksforgeeks.org/",
+      },
+      {
+        name: "ABC",
+        link: "https://www.geeksforgeeks.org/",
+      },
+      {
+        name: "ABC",
+        link: "https://www.geeksforgeeks.org/",
+      },
+    ],
+  },
+  {
+    name: "System Design",
```

Comment:
Since we don't have a System Design as of yet, I suggest we change this to Introduction to Cloud.